blog: pihole (fixes #4)

Liew211 opened the pull request:
https://raw.githack.com/treehouses/treehouses.github.io/blog-pihole/#!pages/blog/2019-07-17-pihole1.md

Liew211 committed:
Create 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Add files via upload

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Add files via upload

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

ArranHL commented:
Might be worth adding a note indicating that  quotation marks should be used around  *"wifiname"* and *"wifipassword"* as this is currently missing.

In addition removing the *bridge* in 
#treehouses *bridge* <wifiname> <hotspotname> [wifipassword] [hotspotpassword] 
would be a good idea if this is not needed.


----------------------------------------------------------------------------------------


As *ping treehouses.local* can also sometimes not return the desired IP address,  an additional line indicating that:

If  *ping treehouses.local* does not return a result, the treehouse IP will have to be found using: 

nmap -sP 192.168.1.*

This will instruct your device to discover and return all the IP's in your immediate vicinity such as active android and apple devices and wifi routers, find one that reads:
treehouses.home

Should the desired result once again not be available, please expand your serch of IP's using:

 nmap -sP 192.


--------------------------------------------------------------------------------


In my case ssh root@ required a password and the only way around this was to utilize ssh pi@  it could be worth mentioning this, depending of whether or not we can confirm it will be an issue long term. 

--------------------------------------------------------------------------------

Great job on the documentation and getting this to work :)

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Delete pihole.png

Liew211 committed:
Add files via upload

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 commented:
@ArranHL I made some changes, and split it up into steps.  I've added in quotes around wifiname and wifipassword, and I included a link on other methods of finding the local IP address.

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Update 2019-07-17-pihole1.md

ArranHL commented:
Awesome stuff! :)

Liew211 committed:
Update 2019-07-17-pihole1.md

Liew211 committed:
Images and blog list pihole

Liew211 committed:
Update 20190717-pihole.md

dogi committed:
Merge branch 'master' into blog-pihole
